src/help/fabs_validations.csv

```diff
@@ -82,8 +82,9 @@ FABS,FABS42.2,"PrimaryPlaceOfPerformanceForeignLocationDescription must be blank
 FABS,FABS43.1,"When PrimaryPlaceOfPerformanceCountryCode is not USA, PrimaryPlaceOfPerformanceCongressionalDistrict must be blank.",Fatal Error,Implemented
 FABS,FABS43.2,"If PrimaryPlaceOfPerformanceZIP+4 is 5-digits or blank, a PrimaryPlaceOfPerformanceCongressionalDistrict must be provided.",Fatal Error,Implemented
 FABS,FABS43.3,"If PrimaryPlaceOfPerformanceCongressionalDistrict is provided, it must be valid in the state or territory indicated by the PrimaryPlaceOfPerformanceCode. For PII-redacted non-aggregate records (i.e., when RecordType = 3), PrimaryPlaceOfPerformanceCongressionalDistrict must be blank.",Fatal Error,Implemented
-FABS,FABS44.1,"If LegalEntityZIPLast4 is not provided and LegalEntityZIP5 is, LegalEntityCongressionalDistrict must be provided.",Fatal Error,Implemented
-FABS,FABS44.2,"If LegalEntityCongressionalDistrict is provided, it must be valid in the 5-digit zip code indicated by LegalEntityZIP5.",Fatal Error,Implemented
+FABS,FABS44.1,"When LegalEntityCountryCode is not USA, LegalEntityCongressionalDistrict must be blank.",Fatal Error,Implemented
+FABS,FABS44.2,"If LegalEntityZIPLast4 is not provided and LegalEntityZIP5 is, LegalEntityCongressionalDistrict must be provided.",Fatal Error,Implemented

```

Comment:
Should the explanation `This does not apply when LegalEntityCountryCode is not USA or when RecordType = 1.` need to be added to 44.2?